src/Kestrel.Transport.Libuv/Internal/LibuvThread.cs

```diff
@@ -48,8 +48,12 @@ public LibuvThread(LibuvTransport transport)
             _log = transport.Log;
             _loop = new UvLoopHandle(_log);
             _post = new UvAsyncHandle(_log);
+
             _thread = new Thread(ThreadStart);
+#if !INNER_LOOP
             _thread.Name = nameof(LibuvThread);
```

Comment:
Why?

Reply:
It was slow on traces, ~25% of LibuvThread starting.

Reply:
That's really too bad. If it wasn't so useful when looking at core dumps, I'd just recommend removing this entirely.